From: Yao Wei (魏銘廷) <mwei@debian.org>
Date: Mon, 8 Apr 2019 00:08:15 +0800
Subject: Remove external fonts in doc
 This patch removes external query to Google Fonts, stops lintian warning
 privacy-breach-generic.
Forwarded: not-needed

---
 documentation/source/_themes/fontPartsTheme/static/fontparts.css | 1 -
 1 file changed, 1 deletion(-)

diff --git a/documentation/source/_themes/fontPartsTheme/static/fontparts.css b/documentation/source/_themes/fontPartsTheme/static/fontparts.css
index 42de4e5..bf4f014 100644
--- a/documentation/source/_themes/fontPartsTheme/static/fontparts.css
+++ b/documentation/source/_themes/fontPartsTheme/static/fontparts.css
@@ -1,5 +1,4 @@
 @charset "UTF-8";
-@import url("https://fonts.googleapis.com/css?family=Source+Code+Pro:400,700|Source+Sans+Pro:400,400i,700");
 /* ---------- resets etc ---------- */
 html {
   box-sizing: border-box;
